linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Set SSH MaxSessions limit'
+
+description: |-
+    The <tt>MaxSessions</tt> parameter specifies the maximum number of open sessions permitted
+    from a given connection.  Setting MaxSessions to 1 will effectively disable
+    session multiplexing, whereas setting it to 0 will prevent all
+    shell, login and subsystem sessions while still permitting forwarding.  The default is 10.
+    Add or correct the following lines to 4 or less. To set MaxSessions edit
```

Comment:
> Add or correct the following lines to 4 or less.

This is specific for the CIS profile, so it shouldn't be part of rule description.

Reply:
Resolved in 9800aa6